src/plugins/table/table-display.js

```diff
@@ -1,13 +1,13 @@
 import React, { useMemo } from 'react';
+import { mapCellsNested } from './table-utils.js';
 import Markdown from '../../components/markdown.js';
-import { mapTwoDimensionalArray } from './table-utils.js';
 import { sectionDisplayProps } from '../../ui/default-prop-types.js';
 
 function TableDisplay({ content }) {
   const { rowCount, columnCount, cells, renderMedia } = content;
 
   const rows = useMemo(() => {
-    const fullCellMap = mapTwoDimensionalArray(rowCount, columnCount, () => null);
+    const fullCellMap = mapCellsNested(rowCount, columnCount, () => null);
```

Comment:
I'm having a bit of a hard time understanding the "mapCellsFlat" and "mapCellsNested" ideas, I don't quite understand how they are mapped nested or flat

Reply:
💡 Ok, so they are both the methods that basically create the table, either flat or normal. 
How about something like:
- "createFlatTable({ rowCount, columnCount, createCell })"
- "createNormalTable({ rowCount, columnCount, createCell })"
to help understand the code easier?

Reply:
As the term "table" refers already to a bigger model (that does not only contain the cells collection), I went for `createTableCellsFlat` and `createTableCellsInRows`. What do you think?

Reply:
Much clearer, thanks. The "map" part of it was confusing, especially since it wan't actually mapping but creating. Maybe another small thing, we could rename the 3rd argument from `mappingFunction` to `createCell`?

Reply:
Yeah, that should have been the logical consequence in the first place :roll_eyes: Renamed it, thanks!